cmd/crc/cmd/start.go

```diff
@@ -60,6 +60,9 @@ func runStart(arguments []string) (*machine.StartResult, error) {
 	checkIfNewVersionAvailable(config.Get(cmdConfig.DisableUpdateCheck).AsBool())
 
 	if err := preflight.StartPreflightChecks(config); err != nil {
+		if err := segmentClient.Upload(err); err != nil {
+			return nil, err
+		}
```

Comment:
This looks like this should not be done in `runStart()`, but in a function which calls `runStart()`, as this has to be duplicated for all error paths happening in `runStart()`